deploy/examples/csi/cephfs/pvc.yaml

```
@@ -3,6 +3,8 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: cephfs-pvc
+  labels:
+    group: test
```

How about label the example so it's clearly related to snapshots?
```suggestion
    group: snapshot-test
```